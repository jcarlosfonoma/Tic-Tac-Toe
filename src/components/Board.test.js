import { cleanup, render } from "@testing-library/react";
import Board from "./Board";

describe("Board component", () => {
  beforeEach(cleanup);

  it("Board should take a snapshot", () => {
    const squares = [null, null, null];
    const { asFragment } = render(<Board squares={squares} />);
    expect(asFragment(<Board />)).toMatchSnapshot();
  });
});
