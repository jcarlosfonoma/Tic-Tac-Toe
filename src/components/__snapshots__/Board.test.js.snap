// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board component Board should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="board-row"
    >
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
    </div>
    <div
      class="board-row"
    >
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
    </div>
    <div
      class="board-row"
    >
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
      <button
        class="sc-bdvvtL fPtRnW"
        data-testid="square-btn"
      />
    </div>
  </div>
</DocumentFragment>
`;
